overlord/snapstate/snapstate.go

```diff
@@ -2016,10 +2061,16 @@ func autoRefreshPhase1(ctx context.Context, st *state.State) ([]string, []*state
 
 	// return all names as potentially getting updated even though some may be
 	// held.
-	names := make([]string, len(updates))
+	var names []string
```

Comment:
I suppose we could have kept/used:

names := make([]string, 0, len(updates))

instead, that's still covers the common case

Reply:
Done.